overrides/scripts/villager_trades_removed.zs

```diff
@@ -0,0 +1,128 @@
+}
+else
+{
+    return new MerchantOffer(<item:minecraft:emerald> * (random.nextInt(10)+15), <item:minecraft:book>, <item:minecraft:enchanted_book>.withTag({StoredEnchantments: [{lvl: 1, id: "dungeons_gear:busy_bee"}]}), 16, 0, 5) as MerchantOffer;
+}
+});
+
+val villagerPro = [
+<profession:blue_skies:alchemist>,
+<profession:blue_skies:nightwatcher>,
+<profession:blue_skies:shoveler>,
+<profession:blue_skies:stargazer>,
+<profession:blue_skies:summoner>,
+<profession:blue_skies:wrangler>,
+<profession:domesticationinnovation:animal_tamer>,
+<profession:friendsandfoes:beekeeper>,
+<profession:minecraft:armorer>,
+<profession:minecraft:butcher>,
+<profession:minecraft:cartographer>,
+<profession:minecraft:cleric>,
+<profession:minecraft:farmer>,
+<profession:minecraft:fisherman>,
+<profession:minecraft:fletcher>,
+<profession:minecraft:leatherworker>,
+<profession:minecraft:librarian>,
+<profession:minecraft:mason>,
+<profession:minecraft:nitwit>,
+<profession:minecraft:none>,
+<profession:minecraft:shepherd>,
+<profession:minecraft:toolsmith>,
+<profession:minecraft:weaponsmith>
+];
+
+for prof in villagerPro 
+{
+    villagerTrades.removeTradesSelling(prof, 1, <item:minecraft:emerald>);
+}
+
+val toBuy = [<item:farmersdelight:barbecue_stick>,
+       <item:farmersdelight:baked_cod_stew>,
+       <item:farmersdelight:beef_stew>,
+       <item:farmersdelight:bone_broth>,
+       <item:farmersdelight:cabbage_rolls>,
+       <item:farmersdelight:chicken_soup>,
+       <item:farmersdelight:cod_roll>,
+       <item:farmersdelight:dog_food>,
+       <item:farmersdelight:egg_sandwich>,
+       <item:farmersdelight:fish_stew>,
+       <item:farmersdelight:fruit_salad>,
+       <item:farmersdelight:kelp_roll>,
+       <item:farmersdelight:kelp_roll_slice>,
+       <item:farmersdelight:shepherds_pie>,
+       <item:farmersdelight:pumpkin_soup>,
+       <item:farmersdelight:salmon_roll>,
+       <item:farmersdelight:stuffed_potato>,
+       <item:farmersdelight:vegetable_noodles>,
+       <item:farmersdelight:vegetable_soup>,
+       <item:farmersdelight:bacon_and_eggs>,
+       <item:farmersdelight:bacon_sandwich>,
+       <item:farmersdelight:stuffed_pumpkin>,
+       <item:farmersdelight:chicken_sandwich>,
+       <item:farmersdelight:dumplings>,
+       <item:farmersdelight:fried_rice>,
+       <item:farmersdelight:glow_berry_custard>,
+       <item:farmersdelight:grilled_salmon>,
+       <item:farmersdelight:hamburger>,
+       <item:farmersdelight:mutton_wrap>,
+       <item:farmersdelight:noodle_soup>,
+       <item:farmersdelight:pasta_with_meatballs>,
+       <item:farmersdelight:pasta_with_mutton_chop>,
+       <item:farmersdelight:roast_chicken>,
+       <item:farmersdelight:roasted_mutton_chops>,
+       <item:farmersdelight:squid_ink_pasta>,
+       <item:farmersdelight:steak_and_potatoes>,
+       <item:farmersdelight:honey_glazed_ham>];
+
+for food in toBuy
+{
```

Comment:
you may see differing opinions but I always prefer opening brackets on same line. 

```js
for food in toBuy {
  // your code
}
```